travis-ci/cleanup-integration-tests.sh

```diff
@@ -15,10 +15,17 @@ if [ -z "$DEPLOYMENT" ]; then
 fi
 export DEPLOYMENT
 
+# This should be able to go away once latest is released
+if [ "$USE_NPM_PACKAGES" = "true" ]; then
```

Comment:
Do we need this so that this step can use the `concurrency.Mutex` from the pacakage?

Reply:
yes.